README.md

```diff
@@ -146,3 +146,32 @@ apl
 
 * tplgtool.py
 <br> Dumps info from tplg binary file.
+
+
+## System configuration tips
+
+### sudo noise
+
+sof-test uses sudo a lot which creates a lot of noise in `journalctl`
+output which is especially a problem when testing. To turn off that
+noise first run `sudo visudo` and add the following line:
+
+```
+Defaults:your_unix_id,root !log_allowed
```

Comment:
your_unix_id and below your_id, can you make them as same? and all CAPITAL? Like YOUR_ID. I was about to copy as it is. :)
Thank you for the tip.

Reply:
And we have env-check.sh. Practical version of README. Can you add this description to it?(only help text)
Finally can we run this by a script? I would want to deploy this by default for SOF CI devices.

Reply:
> your_unix_id and below your_id, can you make them as same? and all CAPITAL? Like YOUR_ID.

Done.

> And we have env-check.sh. Practical version of README.

There's some overlap but I don't think these have the same role.

> Can you add this description to it?(only help text)

I would assume that someone who runs `env-check.sh` has already read the `README.md`. I don't think we should cater for people who don't read the `README.md` first, otherwise we may end up with a `README.1st` that says please read the `README.md` file?

> Finally can we run this by a script? 

Yes, that would be awesome but that's a lot more work and figuring out and testing this already took me more than 1 hour. BTW we should also stop re-inventing Ansible.

> I would want to deploy this by default for SOF CI devices.

Only after the corresponding has been locally and successfully tested  by developers for some time of course :-)